stories/2024-01-01-Eurovision/story.ipynb

```diff
@@ -1238,6 +1238,10 @@
     "#| code-fold: true\n",
     "import plotly.express as px\n",
     "\n",
+    "# The following lines are needed for Quarto to render the interactive plot\n",
+    "import plotly.io as pio\n",
+    "pio.renderers.default = \"notebook\"\n",
```

Comment:
```suggestion
    "pio.renderers.default = \"plotly_mimetype+notebook_connected\"\n",
```
With this change I find it shows beautifully in Qarto, but then no longer shows in Jupyter. With the above (taken from [the docs](https://quarto.org/docs/interactive/widgets/jupyter.html#plotly)) I get it working in both. Does this affect VS Code though? Or maybe this is just a glitch on my system?

Reply:
Works on VSCode